python/py-terminaltables/Portfile

```diff
@@ -15,7 +15,8 @@ description         Generate simple tables in terminals from a nested list of st
 long_description    Easily draw tables in terminal/console applications from \
                     a list of lists of strings. Supports multi-line rows.
 
-python.versions     37 38 39 310
+python.rootname     terminaltables
+python.versions     37 38 39 310 311
 
```

Comment:
the fact that the project uses a `pyproject.toml` file means the port should likely set `python.pep517 yes` and is poetry is used as backend then as well `python.pep517_backend poetry`.

Reply:
Understood, thanks.